src/linagora.esn.calendar/app/components/mini-calendar/mini-calendar.controller.spec.js

```diff
@@ -138,7 +138,7 @@ describe('The miniCalendarController controller', function() {
   }));
 
   afterEach(function() {
-    $scope.$destroy();
+    $scope && $scope.destroy && $scope.$destroy();
```

Comment:
Shouldn't it be:

```suggestion
    $scope && typeof $scope.$destroy === 'function' && $scope.$destroy();
```

By the way why do we need this check?

Reply:
Thank you. This check was implemented because some of the test cases failed to execute $scope.$destroy(). I believe they cannot find `$scope` since it has been removed / destroyed at the end of the test.